dts/bindings/sensor/microcrystal,rv3032-temp.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2025 Baylibre SAS
+# SPDX-License-Identifier: Apache-2.0
+
+description: RV3032C7 temperature sensor
+
+compatible: "microcrystal,rv3032-temp"
+
+include: sensor-device.yaml
+
+properties:
+  low-temp:
+    type: int
+    default: -40
```

Comment:
Need to explain in the description why the default value was selected
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#dt-bindings-default-rules